CHANGELOG.md

```
@@ -80,6 +80,58 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
+- `ProcessingWorkflow WorkflowUtil.parseWorkflow(InputStream, String)`<br>
+  - replace by `ProcessingWorkflow WorkflowUtil.parseWorkflow(List, String)`
+
+### 🔒 Security
+- Fix security warning on `GitHub Actions` of this project
+
+### 📦 Updates
+- pom.xml:
+  - Update dependency `com.google.guava:guava:33.4.0-jre` -> `com.google.guava:guava:33.5.0-jre`
+
+### 🚦 Tests
+  - fix test data resource names for `testValidate_error(String workflowName)`
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)